### PR TITLE
加入中文资源，删除BreadcrumbItem中IsSelected的绑定，并在当前页变化时处理前景色的变化

### PR DESCRIPTION
**Pull request by hubo0831**

### CODE REVIEW

File: src/Orc.Wizard/Orc.Wizard.NET40/Orc.Wizard.NET40.csproj [hunk 1]
```diff
@@ -131,6 +131,9 @@
     <EmbeddedResource Include="..\Orc.Wizard.Shared\Properties\Resources.ru.resx">
       <Link>Properties\Resources.ru.resx</Link>
     </EmbeddedResource>
+    <EmbeddedResource Include="..\Orc.Wizard.Shared\Properties\Resources.zh-Hans.resx">
+      <Link>Properties\Resources.ru.resx</Link>
```
Comment (GeertvanHorrik) [inline src/Orc.Wizard/Orc.Wizard.NET40/Orc.Wizard.NET40.csproj]: Resources.ru.resx must point to zh-Hans instead

File: src/Orc.Wizard/Orc.Wizard.NET45/Orc.Wizard.NET45.csproj [hunk 2]
```diff
@@ -109,6 +109,9 @@
     <EmbeddedResource Include="..\Orc.Wizard.Shared\Properties\Resources.ru.resx">
       <Link>Properties\Resources.ru.resx</Link>
     </EmbeddedResource>
+    <EmbeddedResource Include="..\Orc.Wizard.Shared\Properties\Resources.zh-Hans.resx">
+      <Link>Properties\Resources.ru.resx</Link>
```
Comment (GeertvanHorrik) [inline src/Orc.Wizard/Orc.Wizard.NET45/Orc.Wizard.NET45.csproj]: Resources.ru.resx must point to zh-Hans instead

File: src/Orc.Wizard/Orc.Wizard.Shared/Controls/BreadcrumbItem.xaml [hunk 3]
```diff
@@ -7,27 +7,27 @@
...
+    <Grid ToolTip="{Binding Description}" MinWidth="100" MaxWidth="100" Margin="0 0 0 8">
+      <StackPanel>
+        <Grid Margin="5">
+          <Ellipse x:Name="ellipse" Width="26" Height="26" StrokeThickness="0" Fill="{StaticResource AccentColorBrush4}"
+                   HorizontalAlignment="Center" VerticalAlignment="Center" />
+          <TextBlock Text="{Binding Number}" HorizontalAlignment="Center" Foreground="White"
+                   VerticalAlignment="Center" FontWeight="Medium" FontSize="13" Margin="0,-2,0,0" />
         </Grid>
 
+        <TextBlock Name="txtTitle" Text="{Binding Title}" FontSize="14" HorizontalAlignment="Center"
```
Comment (GeertvanHorrik) [inline src/Orc.Wizard/Orc.Wizard.Shared/Controls/BreadcrumbItem.xaml]: Why did you decide not to use a converter?

File: src/Orc.Wizard/Orc.Wizard.Shared/Controls/BreadcrumbItem.xaml.cs [hunk 4]
```diff
@@ -92,17 +92,18 @@ private void UpdateSelection(bool isSelected)
...
             {
-                var colorAnimation = new ColorAnimation(fromColor, (Color) targetColor, WizardConfiguration.AnimationDuration);
+                var colorAnimation = new ColorAnimation(fromColor, (Color)targetColor, WizardConfiguration.AnimationDuration);
                 Storyboard.SetTargetProperty(colorAnimation, new PropertyPath("Fill.(SolidColorBrush.Color)"));
 
                 storyboard.Children.Add(colorAnimation);
             }
 
             storyboard.Begin(ellipse);
+            this.txtTitle.Foreground = isSelected ? Brushes.Black : Brushes.DimGray;
```
Comment (GeertvanHorrik) [inline src/Orc.Wizard/Orc.Wizard.Shared/Controls/BreadcrumbItem.xaml.cs]: Why did you decide not to use a converter?

File: src/Orc.Wizard/Orc.Wizard.NET40/Orc.Wizard.NET40.csproj (hunk 1 above)
Comment (hubo0831) [inline src/Orc.Wizard/Orc.Wizard.NET40/Orc.Wizard.NET40.csproj]: I changed it;

File: src/Orc.Wizard/Orc.Wizard.NET45/Orc.Wizard.NET45.csproj (hunk 2 above)
Comment (hubo0831) [inline src/Orc.Wizard/Orc.Wizard.NET45/Orc.Wizard.NET45.csproj]: I changed it;

**Comment by hubo0831** [comment]
Dependency property "IsSelected" is not defined, and also, change Foreground of title is same as   changing ellipse style;

**Comment by GeertvanHorrik** [comment]
Thanks for the update, this looks really good. I am not sure if I understand your latest comment. Does it apply to this PR?

**Pull request merged**